### ISSUE TITLE
Created Image with water mark

### ISSUE THREAD

**Issue by fdchiu** [liuliu/swift-diffusion#14]
I have seen it very often the image has watermark like texture on the image. Sometime it's text like texture. Some other time the image is composed of several sub-images. All with the same configuration: model, model parameters, steps, prompts. 
![IMG_1737](https://user-images.githubusercontent.com/5524561/204668904-2150df3f-a98c-4006-95cf-d295d2ede610.PNG)
![IMG_1739 2](https://user-images.githubusercontent.com/5524561/204668924-251e37f9-363c-4b7d-9612-5b7e14277cef.PNG)
![IMG_1741](https://user-images.githubusercontent.com/5524561/204668932-7d3a0dec-4c34-4bf4-bf2d-7505e2481960.PNG)
![IMG_1742](https://user-images.githubusercontent.com/5524561/204668940-29fee704-bd23-4396-905f-550946382cb6.PNG)

**Comment by liuliu** [comment]
Yes. I normally just use inpainting model to remove these. It would be nice if someone integrated a detector with inpainting to do this automatically.

**Comment by fdchiu** [comment]
Let me know if I can help! And how
